src/tests/test_properties/test_properties.cpp

```diff
@@ -1273,12 +1273,12 @@ private Q_SLOTS:
         QCOMPARE(p.anim_get_color("key", 50).g, 255);
         QCOMPARE(p.anim_get_color("key", 50).b, 171);
         QCOMPARE(p.anim_get_color("key", 50).a, 223);
-        QCOMPARE(p.anim_get_color("key", 60).a, 237);
-        QCOMPARE(p.anim_get_color("key", 70).a, 239);
+        QCOMPARE(p.anim_get_color("key", 60).a, 223);
```

Comment:
I think this is a great example of unexpected behavior for the existing implementation. The keyframe on both sides of these frames has a value of 223. But the algorithm would add "ripple" and interpolate different values for intermediate frames.